tox.ini

```diff
@@ -1,14 +1,16 @@
 [tox]
 envlist=
-    py{35,36,37,38,39,310}-{test,celery4,requests,flask,tornado,wsgi,bottle},
-    py{36,37,38,39,310}-{asgi,celery5},
+    py{35,36,37,38,39,310,311}-{test,requests,flask,tornado,wsgi,bottle}
+    py{35,36,37,38,39,310}-celery4
+    py{36,37,38,39,310}-{asgi,celery5}
```

Comment:
Celery doesn't support 3.11 yet as it wraps some of Python's internal traceback-related types and a new method was added: https://github.com/celery/billiard/issues/377